values.yaml

```diff
@@ -152,26 +152,21 @@ coderd:
       key: "password"
 
   # coderd.affinity -- Allows specifying an affinity rule for the `coderd`
-  # deployment. If you set `coderd.replicas` to a value higher than 1, you
-  # may wish to also add a pod anti-affinity definition to ensure that
-  # multiple coderd pods are not scheduled on the same node.
-  affinity: {}
-  # Example:
-  # ```
-  # affinity:
-  #   podAntiAffinity:
-  #     preferredDuringSchedulingIgnoredDuringExecution:
-  #       - podAffinityTerm:
-  #           labelSelector:
-  #             matchExpressions:
-  #               - key: app
-  #                 operator: In
-  #                 values:
-  #                   - "coderd"
-  #           topologyKey: kubernetes.io/hostname
-  #         weight: 1
-  # ```
-
+  # deployment. The default rule attempts to schedule multiple coderd
+  # pods on different nodes. This only really applies if you set
+  # `coderd.replicas` to a value higher than 1.
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app
```

Comment:
I think we should use `app.kubernetes.io/name` instead, the `app` label was a legacy one and the former is a de-facto standard label, as it's listed as recommended https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels